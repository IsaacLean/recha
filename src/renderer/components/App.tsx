import React from 'react'

import AddTodo from '../containers/AddTodo'
import Footer from './Footer'
import VisibleTodoList from '../containers/VisibleTodoList'

const App = (): JSX.Element => (
  <div>
    <AddTodo />
    <VisibleTodoList />
    <Footer />
  </div>
)

export default App
