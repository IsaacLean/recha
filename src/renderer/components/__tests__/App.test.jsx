import Adapter from 'enzyme-adapter-react-16'
import Enzyme, { shallow } from 'enzyme'
import React from 'react'

import App from '../App'

Enzyme.configure({ adapter: new Adapter() })

describe('App', () => {
  it('renders properly', () => {
    const component = shallow(<App />)
    expect(component).toMatchSnapshot()
  })
})
