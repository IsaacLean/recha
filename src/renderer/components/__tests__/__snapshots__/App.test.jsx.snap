// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders properly 1`] = `
<div>
  <Connect(AddTodo) />
  <withRouter(Connect(VisibleTodoList)) />
  <Footer />
</div>
`;
